src/afrey/boot_asset_fingerprint.clj

```diff
@@ -27,8 +36,9 @@
                           (core/output-files)
                           (core/by-ext (or extensions [".html"])))
             file-hashes (into {}
-                          (map (juxt :path :hash))
-                          (core/output-files fileset))]
+                              (map (juxt :path :hash))
+                              (core/by-ext (or rename-extensions [".js" ".css"]) (core/output-files fileset)))
```

Comment:
This implementation calls `core/mv` https://github.com/AdamFrey/boot-asset-fingerprint/pull/3/files/4431d63d72e46d6f39a50550be4a188bff394698#diff-62eeaa3ab1757be261299d76e96ebb0fR62 on the files in the hash, In order to avoid accidental fingerprinting of files such as a `.clj` the `rename-extensions` made sense.